content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -162,7 +162,7 @@ If specified, adds all of the specified tags to all generated spans.
 
 `DD_TRACE_HEADER_TAGS`
 : **TracerSettings property**:`HeaderTags` <br>
-Accepts a map of case-insensitive header keys to tag names and automatically applies matching header values as tags on root spans. Also accepts entries without a specified tag name. <br>
+Accepts a comma-separate list of maps of case-insensitive header keys to tag names and automatically applies matching header values as tags on root spans. Also accepts entries without a specified tag name. <br>
```

Comment:
Isn't it a comma-separated list of key-value pairs?
```suggestion
Accepts a comma-separated list of key-value pairs of case-insensitive header keys to tag names and automatically applies matching header values as tags on root spans. Also accepts entries without a specified tag name. <br>
```

Reply:
Yep, makes sense 👍